state/session/service.js

```diff
@@ -1,27 +1,46 @@
 import { createLogic } from 'redux-logic';
+import cookie from 'cookie_js';
 
 import { API_KEY } from '../../constants';
-import { SESSION_REQUEST_TOKEN } from './types';
-import { sessionRequestTokenSuccess } from './actions';
+import { REQUEST_SESSION } from './types';
+import { requestSessionSuccess } from './actions';
 
-const sessionRequestTokenLogic = createLogic({
-  type: SESSION_REQUEST_TOKEN,
+/* eslint-disable camelcase */
```

Comment:
Use disabling linter near the line with "bad" code, because now this rule disables camelcase to end of file. You can use `eslint-disable-next-line` or `eslint-disable-line`.
https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments

Reply:
Thanks. Fixed